temporalio/client.py

```diff
@@ -6685,6 +6698,33 @@ async def get_worker_task_reachability(
         )
         return WorkerTaskReachability._from_proto(resp)
 
+    async def _apply_headers(
```

Comment:
```suggestion
    async def _encode_headers(
```

May need to change the `_apply_headers` module-level package as well